pandas/tests/indexes/datetimes/test_tools.py

```diff
@@ -947,6 +947,9 @@ def test_to_datetime_barely_out_of_bounds(self):
         with pytest.raises(OutOfBoundsDatetime):
             to_datetime(arr)
 
+        with pytest.raises(OutOfBoundsDatetime):
+            to_datetime(arr, format="%Y-%m-%d %H:%M:%S.%f")
+
```

Comment:
Move this out as a separate test and reference the issue number.

Also, check the error here with the `match` parameter in `pytest.raises`.

Reply:
Thanks for the feedback. I'll move it out. However, my main issue here was that this test would pass locally but fail on the CI build (https://travis-ci.org/pandas-dev/pandas/jobs/460708541). My best guess was that it was due to `tslib.pyx` not being recompiled. Any ideas?

Reply:
Your build may not have had this added test, as I'm relatively certain that your change [above](https://github.com/pandas-dev/pandas/pull/23969#discussion_r237154738) was not correct.